### PR TITLE
Add can details

### PR DESCRIPTION
**Pull request by mistyron**
Added details for CAN. 
Mosdt of the docu is from Gord, I've tweaked and optimized it a bit here and there.

### CODE REVIEW

File: content/HowToSetUpSocketCANInterface.md [hunk 1]
```diff
@@ -0,0 +1,369 @@
+<img src="../files/img/2018_MistyWest_LogoCombo_FINAL_RGB.png" alt="MistyWest" width="400"/><div style="text-align: right">back to [index](../README.md)</div>
+
+# How To Set Up Socket CAN Interface
```
Comment (matinlotfali) [inline content/HowToSetUpSocketCANInterface.md:3]: Please edit the header similar to the headers of other pages. We changed it for the Github Pages.

**Review by matinlotfali — CHANGES_REQUESTED** [review]
Please merge the develop branch into this branch and follow the header pattern used in other pages.

File: content/HowToSetUpSocketCANInterface.md (hunk 1 above)
Comment (mistyron) [reply content/HowToSetUpSocketCANInterface.md:3]: Oh I see now, I need a title like::
```
---
title: "How To Set Up Socket CAN Interface"
---
```
and not the logo and dthe link back to the index anymore. I understand that this is for functionality reasons but can we consiiiiiiiiiiiider to leave the logo along with the link in place, too?

Comment (mistyron) [reply content/HowToSetUpSocketCANInterface.md:4]: right! Good catch!

**Review by matinlotfali — APPROVED** [review]

**Pull request merged**